svf-llvm/lib/SVFIRBuilder.cpp

```diff
@@ -1398,7 +1398,29 @@ const Value* SVFIRBuilder::getBaseValueForExtArg(const Value* V)
         }
         if(totalidx == 0 && !SVFUtil::isa<StructType>(value->getType()))
             value = gep->getPointerOperand();
+    } else if (const LoadInst* load = SVFUtil::dyn_cast<LoadInst>(value)) {
```

Comment:
Could you put examples as the comments in this function?

/*
* Example 1: LLVM IR 
* 
* Example 2: LLVM IR
*/
const Value* SVFIRBuilder::getBaseValueForExtArg(const Value* V)

Reply:
Better to make two examples one for the if branch when not dealing with global and the other for the case in the issue. Would be good to also point out which is the value and which is the gep and base corresponding llvm IR to the implementation code